fedimint-core/src/db/mem_impl.rs

```diff
@@ -178,6 +179,18 @@ impl<'a> IDatabaseTransactionOpsCore for MemTransaction<'a> {
 
         Ok(Box::pin(stream::iter(data)))
     }
+
+    async fn raw_find_by_range(&mut self, range: Range<&[u8]>) -> Result<PrefixStream<'_>> {
```

Comment:
I assume it's implicit in the type, but would be nice to document inclusiveness/exclusiveness on both `raw_find_by_range` and `find_by_range`.

Reply:
We seem to document only `raw_`, ATM, so I kept it this way.